requirements-dev-azure.txt

```diff
@@ -0,0 +1,2 @@
+azure-core>=1.17.0 # azure
+azure-storage-blob>=12.8.1 # azure
```

Comment:
Just wanted to doublecheck: would `requirements-dev-azure.txt` need to be added to `requirements-dev.txt`?

Reply:
@alexsherstinsky and @jcampbell ^

Reply:
Good catch! Seems that's consistent with the other optional requirements, adding along with changelog.